docs/scripts/getting-started.md

```diff
@@ -1,98 +1,82 @@
-
-Scripts make it easier for Hummingbot users to prototype new strategies focusing only on the functionality and forgetting about configuration files and compilation
```

Comment:
You shouldn't be overriding this file, since this is a Getting Started guide for scripts generally